release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
-// 4.14.29
+[id="ocp-4-14-30"]
+=== RHSA-2024:3881 - {product-title} 4.14.30 bug fix and security updates
+
+Issued: 2024-06-19
+
+{product-title} release 4.14.30, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:3881[RHSA-2024:3881] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:3918[RHSA-2024:3918] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.30 --pullspecs
+----
+
+[id="ocp-4-14-30-bug-fixes"]
+==== Bug fixes
+
+* Previously,  (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])
+
+* https://issues.redhat.com/browse/OCPBUGS-35034
+
+* Previously, an AWS policy issue prevented the Cluster API Provider for AWS from retrieving the necessary domain information. As a consequence, installing an AWS hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34856[*OCPBUGS-34856*])
+
+* https://issues.redhat.com/browse/OCPBUGS-34814
+
+* https://issues.redhat.com/browse/OCPBUGS-34792 RHE
+
+* https://issues.redhat.com/browse/OCPBUGS-34668
+
+* https://issues.redhat.com/browse/OCPBUGS-34657
+
+* Previously, certain situations caused transfer of an Egress IP address from one node to a different node to fail, and this failure impacted the OVN-Kubernetes network. The network failed to send gratuitous Address Resolution Protocol (ARP) requests to peers to inform them of the new node’s medium access control (MAC) address. As a result, peers would temporarily send reply traffic to an old node and this traffic led to failover issues. With this release, the OVN-Kubernetes network correctly sends a gratuitous ARP to peers to inform them of the new Egress IP node MAC address, so that each peer can send reply traffic to the new node without causing failover time issues. (link:https://issues.redhat.com/browse/OCPBUGS-34570[*OCPBUGS-34570*])
+
+* Previously, the `wait-for-ceo` command used during the bootstrapping process to verify etcd rollout would not report errors for some failure modes. With this release, those error messages should now be visible on the `bootkube` script if the command exits in an error case. (link:https://issues.redhat.com/browse/OCPBUGS-34495[*OCPBUGS-34495*])
+
+* https://issues.redhat.com/browse/OCPBUGS-5943
```

Comment:
* Previously, the DNS Operator did not verify that a cluster had ready nodes with available CPU in at least two availability zones, and the DNS daemon set did not use surge for rolling updates. As a result, clusters in which all nodes were in the same availability zone would repeatedly emit `TopologyAwareHintsDisabled` events for the cluster DNS service. With this release, the `TopologyAwareHintsDisabled` events are no longer emitted on clusters that do not have nodes in multiple availability zones and the issue has been resolved. (link:[https://issues.redhat.com/browse/OCPBUGS-5943[OCPBUGS-5943])